File: swagger/models/get_scheduled_service_earnings_response.py
from .base_model import BaseModel

from .pagination_response import PaginationResponse

class GetScheduledServiceEarningsResponse(BaseModel):
	"""
	no description available

	Attributes:

		pagination_response
		Contains information about the pagination used.

		scheduled_service_earnings
		Contains the class payroll events.
	"""

	input_map = {
		'PaginationResponse': ('pagination_response', PaginationResponse),
		'ScheduledServiceEarnings': 'scheduled_service_earnings',
		}

	output_map = {
		'pagination_response': 'PaginationResponse',
		'scheduled_service_earnings': 'ScheduledServiceEarnings',
		}

	pagination_response = None
	scheduled_service_earnings = None

File: swagger/Models/GetScheduledServiceEarningsResponse.php
<?php

namespace App\MindBody\Models;

/**
 * no description available
 */
class GetScheduledServiceEarningsResponse extends BaseModel
{
	/**
	 * Contains information about the pagination used.
	 * 
	 * @var PaginationResponse
	 */
	public $paginationResponse;
	/**
	 * Contains the class payroll events.
	 * 
	 * @var ScheduledServiceEarningsEvent[]
	 */
	public $scheduledServiceEarnings;

	protected function getInputMap()
	{
		return [
			'PaginationResponse' => ['paginationResponse', PaginationResponse::class],
			'ScheduledServiceEarnings' => ['scheduledServiceEarnings', ScheduledServiceEarningsEvent::class],
			];
	}

	protected function getOutputMap()
	{
		return [
			'paginationResponse' => 'PaginationResponse',
			'scheduledServiceEarnings' => 'ScheduledServiceEarnings',
			];
	}
}
